Dockerfile

```diff
@@ -40,7 +40,7 @@ RUN  chown -R app: /home/app
 ENV HOME /home/app
 WORKDIR /home/app
 ENV APP_HOME /home/app
-USER app
+USER 1000
```

Comment:
🙋‍♂ These changes have now been merged via #114 and can be removed from this pull request.